recipe_scrapers/__init__.py

```diff
@@ -605,22 +606,40 @@ def scraper_exists_for(url_path: str) -> bool:
     return host_name in get_supported_urls()
 
 
+def check_scraper_allowed(url_path, host_name, **options: Any) -> bool:
```

Comment:
can you rename to `robots_txt_says_allowed` or something along that lines. (I'd like to mention robots in the name).

Reason is robots.txt is considered voluntary compliance mechanism which may defer from site's T&C so I want to be explicit with the method name.